### PR TITLE
Create git.md

### PR DESCRIPTION
**Pull request by bognix**

### CODE REVIEW

File: git.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Apprentice aka _My First Commit_
+* I know what is a version control system and what are the use cases for it
+* I know what Git is
+* I can initialize empty Git repository on my local machine
+* I understand different states of files in Git repository
+** untracked
+** unstaged
+** staged
+** commited
+* I can list modified fiels on my local file system in version controled directory
```
Comment (bognix) [inline git.md]: typo

File: git.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+* I can reset modified changes in a file
+* I can stage changes for commit
```
Comment (bognix) [inline git.md]: for a commit

File: git.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+* I can unstage changes
+* I understand the concept of a commit and I can commit changes
+* I can make Git ignore files in the repository
+
+### Intermediate aka _I can branch!_
+* I can answer all the questions from level 1 :)
```
Comment (bognix) [inline git.md]: redundant

File: git.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* I can create a branch locally and switch between the branches
+* I understand the concept of branching and I know which commits / changes I can expect on different branches
+* I know there are remote Git servers and services that offer it like: Github, Bitbucket or Gitlab
+* I can stash my changes and move them between branches
```
Comment (bognix) [inline git.md:29]: those two above the Github, Bitbucket and Gitlab

File: git.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+* I know how to move commits between branches, `git cherry-pick` command
+
+### Advanced aka _Teamplayer_
+* I know I can sync my changes with remote Git server
```
Comment (bognix) [inline git.md]: with a remote

File: git.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+* I can push my changes to the remote server and pull them from it
+* I can create branches locally and push them to the remote server
+* I know what the mysterious `origin` means, I know how to add a remote server, change it's URL or remove it from my local git configuration
+* I can collaborate with my teamates, share code, merge their changes into my branch and resolve the potential confilcts (in code :D)
+* I know different approaches to collaboration in git project. I know what means names like:
```
Comment (bognix) [inline git.md]: I know what following names mean:

**Review by gitkuba — APPROVED** [review]

File: git.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
+# Git
+
+basic shell (command line) usage
+
+## Intro
+Before you jump into Git it's good to first learn what are the version control systems and when it's good to use one.
+You can read more about VCS (shortcut from Version Control System) in this article: https://pl.atlassian.com/git/tutorials/what-is-version-control
```
Comment (rogatty) [inline git.md]:
```suggestion
You can read more about VCS (shortcut from Version Control System) in this article: https://www.atlassian.com/git/tutorials/what-is-version-control
```

File: git.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+## Classification / acceptance questions
+### Apprentice aka _My First Commit_
+* I know what is a version control system and what are the use cases for it
+* I know what Git is
+* I can initialize empty Git repository on my local machine
+* I understand different states of files in Git repository
+** untracked
```
Comment (rogatty) [inline git.md]:
```
  * untracked
```
It's not JIRA 🙂

File: git.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+** unstaged
+** staged
+** commited
+* I can list modified fields on my local file system in version controled directory
```
Comment (rogatty) [inline git.md]: Fields or files?

File: git.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+* I can stage changes for a commit
+* I can unstage changes
+* I understand the concept of a commit and I can commit changes
+* I can make Git ignore files in the repository
+
+### Intermediate aka _I can branch!_
+* I can create a branch locally and switch between the branches
+* I understand the concept of branching and I know which commits / changes I can expect on different branches
+* I can stash my changes and move them between branches
+* I know how to move commits between branches, `git cherry-pick` command
```
Comment (rogatty) [inline git.md]: Move or copy?

File: git.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+### Advanced aka _Teamplayer_
+* I know there are remote Git servers and services that offer it like: Github, Bitbucket or Gitlab
```
Comment (rogatty) [inline git.md]:
```suggestion
* I know there are remote Git servers and services that offer it like: GitHub, Bitbucket or GitLab
```

File: git.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+* I know it's possible to host my own remote Git repository
+* I know I can sync my changes with a remote Git server
+* I can push my changes to the remote server and pull them from it
+* I can create branches locally and push them to the remote server
+* I know what the mysterious `origin` means, I know how to add a remote server, change it's URL or remove it from my local git configuration
```
Comment (rogatty) [inline git.md]:
```suggestion
* I know what the mysterious `origin` means, I know how to add a remote server, change its URL or remove it from my local git configuration
```

File: git.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+* I can collaborate with my teamates, share code, merge their changes into my branch and resolve the potential confilcts (in code :D)
```
Comment (rogatty) [inline git.md]:
```suggestion
* I can collaborate with others, share code, merge their changes into my branch and resolve the potential merge conflicts
```

File: git.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+* I know different approaches to collaboration in git project. I know what following names mean:
+** feature branch
```
Comment (rogatty) [inline git.md]:
```
  * feature branch
```

File: git.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+
+### Advanced aka _Teamplayer_
+* I know there are remote Git servers and services that offer it like: Github, Bitbucket or Gitlab
+* I know it's possible to host my own remote Git repository
+* I know I can sync my changes with a remote Git server
+* I can push my changes to the remote server and pull them from it
+* I can create branches locally and push them to the remote server
+* I know what the mysterious `origin` means, I know how to add a remote server, change it's URL or remove it from my local git configuration
+* I can collaborate with my teamates, share code, merge their changes into my branch and resolve the potential confilcts (in code :D)
+* I know different approaches to collaboration in git project. I know what following names mean:
```
Comment (rogatty) [inline git.md]:
```suggestion
* I know different approaches to collaboration in git project. I know what these mean:
```

File: git.md [hunk 16]
```diff
@@ -0,0 +1,58 @@
...
+* I can revert my changes, both on my local machine and on remote Git server
+* I know how to overwrite the Git history and I promise never to use it! :D
+
+## Sources
+### Books
+- https://git-scm.com/book/en/v2
+
+### Articles
+- https://itnext.io/become-a-git-pro-in-just-one-blog-a-thorough-guide-to-git-architecture-and-command-line-interface-93fbe9bdb395
+- https://pl.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
```
Comment (rogatty) [inline git.md]:
```suggestion
- https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow
```

File: git.md [hunk 17]
```diff
@@ -0,0 +1,58 @@
...
+* I understand the difference between merging and rebasing
+* I can revert my changes, both on my local machine and on remote Git server
+* I know how to overwrite the Git history and I promise never to use it! :D
+
+## Sources
+### Books
+- https://git-scm.com/book/en/v2
+
+### Articles
+- https://itnext.io/become-a-git-pro-in-just-one-blog-a-thorough-guide-to-git-architecture-and-command-line-interface-93fbe9bdb395
```
Comment (rogatty) [inline git.md]: @kvas-damian used `[title](url)` syntax in https://github.com/bethinkpl/bit-docs/pull/3, we should keep it consistent either way.

File: git.md [hunk 18]
```diff
@@ -4,18 +4,18 @@ basic shell (command line) usage
...
 * I can initialize empty Git repository on my local machine
 * I understand different states of files in Git repository
-** untracked
-** unstaged
-** staged
-** commited
+* * untracked
+* * unstaged
+* * staged
+* * commited
```
Comment (rogatty) [inline git.md]: This isn't right, there are double bullets.

**Pull request merged**